src/test/interface/io/json/json_data_test.cpp

```diff
@@ -144,6 +177,18 @@ TEST(ioJson, jsonData_real_array_1D) {
   test_real_var(jdata, txt, "foo", expected_vals, expected_dims);
 }
 
+TEST(ioJson, jsonData_complex_array_1D) {
```

Comment:
[question]
Is there a test that we get the right error message if the variable is not found?

Reply:
We test if the variable name exists in test_complex_var

`EXPECT_EQ(true, (jdata.contains_r(name) || jdata.contains_i(name)));`